deploy/scripts/shared.sh

```
@@ -0,0 +1,73 @@
+        # Ensure quadlet output directory exists
+        mkdir -p "${QUADLET_FILES_OUTPUT_DIR}"
+
+        # Process standalone container template
+        cp "$container_file" "${QUADLET_FILES_OUTPUT_DIR}/flightctl-${service_name}.container"
+    else
+        # Normal mode - process all container files except standalone ones
+        mkdir -p "${QUADLET_FILES_OUTPUT_DIR}"
+
+        for container_file in "${template_dir}/flightctl-${service_name}"/*.container; do
+            if [[ -f "$container_file" ]] && [[ ! "$container_file" == *"-standalone.container" ]]; then
+                local base_filename=$(basename "$container_file")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)